internal/handlers/common/getmore.go

```diff
@@ -81,7 +83,56 @@ func GetMore(ctx context.Context, msg *wire.OpMsg, registry *cursor.Registry) (*
 	}
 
 	// TODO maxTimeMS https://github.com/FerretDB/FerretDB/issues/1808
-	// TODO comment
```

Comment:
I think this todo was to handle `comment` param (e.g. we have it ignored in the other parts of the code). So, as I don't see `comment` being handled here, this TODO seems to be actual (we might need an issue for it).

Reply:
Added issue link, see around L137